notebooks/timeseries.ipynb

```diff
@@ -0,0 +1,568 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# OpenSCM Timeseries\n",
+    "\n",
+    "In OpenSCM, the class for handling timeseries is `openscm.timeseries.Timeseries`. \n",
+    "\n",
+    "A timeseries in OpenSCM is always initialised from arrays i.e. discrete data which are associated with specific points in time. In this notebook we call the initialisation values the 'source data' and the times with which they are associated the 'source times'.\n",
```

Comment:
By definition a timeseries is always discrete...

Reply:
oops